pmm/pmm2-server-release.groovy

```diff
@@ -35,6 +35,9 @@ pipeline {
             defaultValue: '2.0.0',
             description: 'PMM2 Server version',
             name: 'VERSION')
+        string(
+            description: 'Password for PGP key',
+            name: 'PGP_KEY_PASS')
```

Comment:
I am not sure, but why we need password as a job parameter?

Reply:
no

Reply:
It's an old leftover. I've removed password string